tools/comp/markv.cpp

```diff
@@ -158,6 +166,18 @@ int main(int argc, char** argv) {
             return 1;
           } else if (0 == strcmp(argv[argi], "--validate")) {
             validate_spirv_binary = true;
+          } else if (0 == strcmp(argv[argi], "--model=shader_lite")) {
+            if (model_type != spvtools::kMarkvModelUnknown)
+              fprintf(stderr, "error: More than one model specified\n");
```

Comment:
This prints an error but keeps going.   Should instead probably error out from the program.